modules/amazon-s3-configuring-acs.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-amazon-s3.adoc
+:_module-type: PROCEDURE
+[id="amazon-s3-configuring-acs_{context}"]
+= Configuring {product-title}
+
+To configure Amazon S3 backups, create a new integration in {product-title}.
+
+.Prerequisites
+* An existing S3 Bucket.
+To create a new bucket with required permissions, see the official Amazon documentation topic link:https://docs.aws.amazon.com/AmazonS3/latest/user-guide/create-bucket.html[How Do I Create an S3 Bucket?]
+* `Read`, `write`, and `delete` permissions for the S3 bucket, the *Access key ID*, and the *Secret access key*.
+* If you are using *KIAM*, *kube2iam* or another proxy, then an *IAM role* that has the `read`, `write`, and `delete` permissions.
+
+.Procedure
+. On the RHACS portal, navigate to *Platform Configuration* -> *Integrations*.
+. Scroll down to the *External backups* section and select *Amazon S3*.
+. Click *New Integration* (add icon).
+. Enter a name for *Integration Name*.
+. Enter the number of backups to retain in the *Backups To Retain* box.
+. For *Schedule*, select the backup frequency as daily or weekly and the time to run the backup process.
+. Enter the *Bucket* name in which you want to store the backup.
+. Optionally, enter an *Object Prefix* if you want to save the backups in a specific folder structure.
+For more details, see the official Amazon documentation topic link:https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingMetadata.html#object-keys[Object keys].
```

Comment:
Same w/ removing "official", and the page name changed if you don't mind updating